teleport/metadata.csv

```diff
@@ -2,28 +2,28 @@ metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation
 teleport.auth.audit_log.emit_events,count,,,,Number of audit events emitted,0,teleport,,
 teleport.auth.audit_log.failed_disk_monitoring,count,,,,Number of times disk monitoring failed.,0,teleport,,
 teleport.auth.audit_log.failed_emit_events,count,,,,Number of times emitting audit event failed.,0,teleport,,
-teleport.auth.audit_log.parquetlog_age_oldest_processed_message,gauge,,,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_batch_count,count,,,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_batch_processing_seconds.bucket,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_batch_processing_seconds.count,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_batch_processing_seconds.sum,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_batch_size.bucket,count,,,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_batch_size.count,count,,,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_batch_size.sum,count,,,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_delete_events_seconds.bucket,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_delete_events_seconds.count,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_delete_events_seconds.sum,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_errors_from_collect_count,count,,,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_last_processed_timestamp,gauge,,,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_s3_flush_seconds.bucket,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_s3_flush_seconds.count,count,,second,,,0,teleport,,
-teleport.auth.audit_log.parquetlog_s3_flush_seconds.sum,count,,second,,,0,teleport,,
+teleport.auth.audit_log.parquetlog_age_oldest_processed_message,gauge,,,,"Number of age of oldest event in Parquet-format audit log.                                         ",0,teleport,,
```

Comment:
Why these spaces?

Reply:
I'm still working on removing them. I pushed the PR  a bit early for docs review.